styles.css

```diff
@@ -0,0 +1,146 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
```

Comment:
- [ ] While you are using the `font-family: 'Poppins'` it's not being implemented because you need to import this font from [google-fonts](https://fonts.google.com/specimen/Poppins?query=poppins) and put the import on top of your CSS file as 'Poppins' is not a available by default 

```css
@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap');
```

Reply:
- [ ] I noticed you have not applied this change request too. Please note that these changes are requested to help you improve your skills and develop a good-looking project that an be presented to employers when job searching. 🛩️